src/model/src/ASX_DATA_MOD.F

```diff
@@ -261,16 +261,22 @@ Module ASX_DATA_MOD
       Real, allocatable, private :: BUFF2D( :,: )   ! 2D temp var
       Real, allocatable, private :: BUFF3D( :,:,: ) ! 3D temp var
 
-! Canopy option control (Wei Li)
-      CHARACTER( 20 ), SAVE :: CTM_CANOPY_SHADE = 'CTM_CANOPY_SHADE    '! env var for in-line
+! Canopy option control
+      CHARACTER( 20 ), SAVE :: CTM_CANOPY_SHADE = 'CTM_CANOPY_SHADEC'! env var for in-line
```

Comment:
Why the additional letter "C" is added?
TM_CANOPY_SHADE = 'CTM_CANOPY_SHADE '
vs
TM_CANOPY_SHADE = 'CTM_CANOPY_SHADEC'

Reply:
corrected.